api/monitoring/alerts.js

```
@@ -0,0 +1,476 @@
+          return res.status(401).json({ error: 'Unauthorized' });
+        }
+        
+        const { action, ...body } = req.body;
+        
+        switch (action) {
+          case 'test':
+            // Test alert configuration
+            const testResult = await testAlertConfiguration(
+              alertManager,
+              body.channel || 'all'
+            );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)